Update xml-schema-definition-tool-xsd-exe.md (#29463)
dotnet/docs: docs/standard/serialization/xml-schema-definition-tool-xsd-exe.md

|**/e\[lement\]:**_element_|Specifies the element in the schema to generate code for. By default all elements are typed. You can specify this argument more than once.|
|**/enableDataBinding**|Implements the <xref:System.ComponentModel.INotifyPropertyChanged> interface on all generated types to enable data binding. The short form is `/edb`.|
|**/enableLinqDataSet**|(Short form: `/eld`.) Specifies that the generated DataSet can be queried against using LINQ to DataSet. This option is used when the /dataset option is also specified. For more information, see [LINQ to DataSet Overview](../../framework/data/adonet/linq-to-dataset-overview.md) and [Querying Typed DataSets](../../framework/data/adonet/querying-typed-datasets.md). For general information about using LINQ, see [Language-Integrated Query (LINQ) - C#](../../csharp/programming-guide/concepts/linq/index.md) or [Language-Integrated Query (LINQ) - Visual Basic](../../visual-basic/programming-guide/concepts/linq/index.md).|

> |**/f\[ields\]**|Generates fields only. By default, get / set properties are generated in addition to the backing fields.|

Reviewer: `get` and `set` are methods, not properties. Tweaked wording and added a backlink.

```suggestion
|**/f\[ields\]**|Generates fields only. By default, [properties with backing fields](https://docs.microsoft.com/dotnet/csharp/programming-guide/classes-and-structs/properties#properties-with-backing-fields).|
```

> |**/f\[ields\]**|Generates fields only. By default, [properties with backing fields](https://docs.microsoft.com/dotnet/csharp/programming-guide/classes-and-structs/properties#properties-with-backing-fields) are generated.|

Reviewer 2: ```suggestion
|**/f\[ields\]**|Generates fields only. By default, [properties with backing fields](../../csharp/programming-guide/classes-and-structs/properties.md#properties-with-backing-fields) are generated.|
```

Author: @tdykstra I don't know what the standard is for relative vs. absolute URLs. But there is no ".md" extension on the live website. You get 404s.

Reviewer 2: The docs build will translate the .md file reference to a docs URL.
